manifests/compose/compose.mock-acpi.yaml

```diff
@@ -0,0 +1,84 @@
+services:
+  ### 📦 kepler created from the current repo (local development)
+  kepler-dev:
+    build:
+      context: ../../
+      dockerfile: build/Dockerfile
+    ports:
+      # NOTE: use 9888 to keep the host 8888 port free for any local testing
+      # e.g. sudo ./bin/kepler 
+      - "9188:8888"
+    privileged: true
+    pid: host
+
+    volumes:
+      - type: bind
+        source: /proc
+        target: /proc
+      - type: bind
+        source: /sys
+        target: /sys
+      - type: bind
+        source: ./mock-acpi/kepler-config/etc/kepler
+        target: /etc/kepler
+
+      # NOTE: use the weights from the local repo
+      - type: bind
+        source: ../../data/model_weight/
+        target: /var/lib/kepler/data
+      - type: bind
+        source: ../../data/cpus.yaml
+        target: /var/lib/kepler/data/cpus.yaml
+      - "acpi-mock:/var/acpi-mock:z"
+
+    entrypoint: [/usr/bin/bash, -c]
+
+    command:
+      - |
+        echo Starting kepler;
+        # NOTE: uncomment to wait for estimator container
+        # echo waiting for estimator socket to be ready;
+        # until [[ -e /tmp/estimator.sock ]]; do
+        #   echo " ... waiting for socket";
+        #   sleep 1;
+        # done;
+        set -x;
+        /usr/bin/kepler \
+          -address "0.0.0.0:8888" \
+          -v "3" \
```

Comment:
usually it's 1 for normal test and 5 for debug
what's the reason to set 3 here?

Reply:
no specific reason, this not hard coded in code, we can change it when we run the test.